Base/usr/share/man/man1/bt.md

```diff
@@ -0,0 +1,44 @@
+## Name
+
+bt - View the backtrace of the specified process id
```

Comment:
```suggestion
bt - view the backtrace of the specified process
```

(lowercase just to match existing pattern)

Reply:
Fixed, thanks!